tldw_Server_API/app/core/AuthNZ/repos/api_keys_repo.py

```
@@ -0,0 +1,837 @@
+                            scope,
+                            expires_at.isoformat() if expires_at else None,
+                            rate_limit,
+                            json.dumps(allowed_ips) if allowed_ips else None,
+                            json.dumps(metadata) if metadata else None,
+                        ),
+                    )
+                    key_id = getattr(cursor, "lastrowid", None)
+                    try:
+                        await conn.commit()
+                    except Exception:
+                        pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)